Documentation/ApiOverview/DependencyInjection/Index.rst

```diff
@@ -471,110 +471,113 @@ Configure dependency injection in extensions
 
 Extensions have to configure their classes to make use of the
 dependency injection. This can be done in :file:`Configuration/Services.yaml`.
-Alternatively :file:`Configuration/Services.php` can be used.
+Alternatively, :file:`Configuration/Services.php` can also be used.
 A basic :file:`Services.yaml` file of an extension looks like the following.
 
-*Note:*
-Whenever service configuration or class dependencies change, the Core cache needs
-to be flushed in the Install Tool to rebuild the compiled Symfony container.
-Flushing all caches from the cache clear menu does not flush the compiled Symfony container.
+..  note::
+    Whenever the service configuration or class dependencies change, the Core
+    cache must be flushed in the Install Tool or via the :ref:`CLI <cli-mode>`
+    command `cache:flush` to rebuild the compiled Symfony container. Flushing
+    all caches from the "Clear cache" menu does not flush the compiled Symfony
```

Comment:
```suggestion
    all caches from the :guilabel:`Clear cache` menu does not flush the compiled Symfony
```

Reply:
Yes thought also about it. But there is no menu labeled "Clear cache" just the flash icon, so guilabel is IMHO not right.

Reply:
But the icon has the title and therefore does have the name?